### PR TITLE
codeintel: executor setup walkthrough docs

### PR DESCRIPTION
**Pull request by chrismwendt**
This a step-by-step walkthrough of setting up a single executor on GCP.

I put this at the top of `enable_auto_indexing.md`, but we can change/move this elsewhere if we want.

Resolves https://github.com/sourcegraph/sourcegraph/issues/27864

### CODE REVIEW

**Comment by mrnugget** [comment]
> I put this at the top of `enable_auto_indexing.md`, but we can change/move this elsewhere if we want.

I think it would be best if this was under `Installation` here: https://docs.sourcegraph.com/admin/deploy_executors

A new section, maybe: `Example installation` that contains all of the steps here.

File: doc/code_intelligence/how-to/enable_auto_indexing.md [hunk 1]
```diff
@@ -6,6 +6,82 @@
...
+```
+
+2. Install [`gcloud`](https://cloud.google.com/sdk/docs/install)
+3. Run `gcloud auth application-default login`
+4. Open your Sourcegraph instance in your browser, click your profile in the top right, click **Site admin**, expand **Configuration**, click **Site configuration**, and set:
+  - `"externalURL": "<URL>"` to a URL that is accessible from the GCP VM that will be created later (e.g. a public URL such as `https://sourcegraph.acme.com`)
+  - `"executors.accessToken": "<new long secret>"` to a new long secret (e.g. `cat /dev/random | base64 | head -c 20`)
+  - `"codeIntelAutoIndexing.enabled": true`
+5. Download the example [`main.tf`](https://github.com/sourcegraph/terraform-google-executors/blob/a0110747f70067b9b0d1c382334de02b86821ae3/examples/single-executor/main.tf) and change these:
+  - `project`: your GCP project name and change `region` and `zone` if needed
```
Comment (mrnugget) [inline doc/code_intelligence/how-to/enable_auto_indexing.md:26]: Invalidated by https://github.com/sourcegraph/terraform-google-executors/commit/15b700fe8a6d909c197df769ac974bb2799892b0

File: doc/code_intelligence/how-to/enable_auto_indexing.md [hunk 2]
```diff
@@ -6,6 +6,82 @@
 <p><b>We'd love to ensure that this feature is meeting the requirements of our users.</b> If you have input or feedback on this feature, you can either <a href="https://about.sourcegraph.com/contact">contact us directly</a>, <a href="https://github.com/sourcegraph/sourcegraph">file an issue</a>, or <a href="https://twitter.com/sourcegraph">tweet at us</a>.</p>
 </aside>
 
+Let's walk through setting up a single executor VM on GCP and indexing a repository.
+
+1. Install Terraform `0.13.7` (must be exact):
```
[the remark this answers is not in the document]
Comment (chrismwendt) [reply doc/code_intelligence/how-to/enable_auto_indexing.md]: Added a link to `.tool-versions` so admins can double check.

**Comment by chrismwendt** [comment]
Updated links and moved to `deploy_executors` under a new section "Example installation".

**Pull request merged**